ansible/playbooks/docker-cleanup.yml

```
@@ -0,0 +1,255 @@
+---
+- name: Docker Cleanup Operations
+  hosts: "{{ target_hosts | default('enabled_hosts') }}"
+  gather_facts: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)